core/templates/pages/story-editor-page/navbar/story-editor-navbar.component.html

```diff
@@ -1,30 +1,36 @@
 <ul class="nav navbar-nav oppia-navbar-nav navbar-desktop float-right">
   <li class="oppia-dropdown-menu-container">
     <div ngbDropdown>
-      <button class="btn btn-light dropdown oppia-save-changes-button float-left e2e-test-save-story-button"
-              [ngClass]="{'btn-success': isStorySaveable()}"
-              (click)="saveChanges()"
-              [disabled]="!isStorySaveable()">
-        <span *ngIf="!isSaveInProgress()">
-          <i class="fas fa-save md-18 md-dark oppia-save-publish-button-icon story-editor-save-draft-icon"
-             alt="Save Story">
-          </i>
-          <span class="oppia-save-publish-button-label oppia-save-draft-text float-left">
-            Save Draft
-          </span>
-          <span class="d-none d-md-block float-left oppia-changes-count"
-                *ngIf="getChangeListLength() > 0">
-            ({{ getChangeListLength() }})
+      <span [matTooltip]="validationIssues.concat(explorationValidationIssues).concat(prepublishValidationIssues).join('\n')"
```

Comment:
This is quite complicated code done in HTML can we do it in the TS file and only use the result here?